pkg/util/kubeobjects/statefulset/builder.go

```diff
@@ -59,11 +60,11 @@ func SetTopologySpreadConstraints(topologySpreadConstraints []corev1.TopologySpr
 	}
 }
 
-func SetAllLabels(appLabels, labels map[string]string) builder.Option[*appsv1.StatefulSet] {
+func SetAllLabels(appLabels *labels.AppLabels, customLabels map[string]string) builder.Option[*appsv1.StatefulSet] {
```

Comment:
I don't like idea to have dependency here of labels package, because if I want just map[string]string this code won't work.